### PR TITLE
Apply [RFC: Universal Asset Notation] standard

### LINKED ISSUE #39: Apply [RFC: Universal Asset Notation] standard
- https://github.com/nervosnetwork/rfcs/pull/335

### PR DESCRIPTION
**Pull request by Flouse**
- resolve https://github.com/nervosnetwork/godwoken-info/issues/39

## Reference
- https://github.com/nervosnetwork/rfcs/pull/335

## Related PR
- https://github.com/nervosnetwork/force-bridge/pull/373

### CODE REVIEW

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
Updated in gwscan v1 https://v1.gwscan.com/tokens/bridge

**Comment by jordanmack** [comment]
@Flouse A few minor issues/suggestions. 

1. IBFR starts with a lowercase "i" in the main list. UAN specifies that the asset symbol must be all capitals. It is ok on the details page.
2. pCKB and TAI do not have a UAN, which is inconsistent with YOK.
3. For aesthetics, it may look better if the UAN is a separate column between Token and Address. Another option is to only include it on the details page and not on the main list at all. UAN is not really designed to be a human-friendly label so it may not be necessary on the main list.
4. I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge.

**Comment by Flouse** [comment]
- [x] pCKB and TAI do not have a UAN, which is inconsistent with YOK
- [x] I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge.

**Comment by Flouse** [comment]
> 4\. I believe the UAN paths being used are correct for assets that still reside on L1 before being moved to Godwoken. For example, "1INCH.ckb|fb.eth" is the 1INCH token on CKB L1, which came from Ethereum via Force Bridge. If this list is reflecting tokens that currently are on Godwoken from L1, then the path would be 1INCH.gw|gw.ckb|fb.eth; 1INCH token on Godwoken, which came from CKB L1 via Godwoken Bridge, which came from Ethereum via Force Bridge. (by jordanmack)

Plan to name Godwoken Bridge as `gb` distinguishing to `gw`, so `1INCH` on GwScan will look like:
`1INCH.gw|gb.ckb|fb.eth`

> https://github.com/nervosnetwork/rfcs/pull/335#discussion_r911574244

@jordanmack what do you think?

**Comment by jordanmack** [comment]
> Plan to name Godwoken Bridge as `gb` distinguishing to `gw`, so `1INCH` on GwScan will look like:
> `1INCH.gw|gb.ckb|fb.eth`

Yes, I think that is a good idea. I see it used as "gw" and "gb" in different places. Using "gb" is probably the intended.

**Review by jjyr — APPROVED** [review]

**Comment by Keith-CY** [comment]
I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

**Comment by Flouse** [comment]
> I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

I think this slight difference won't cause confuse, so let‘s approve this PR and merge?
@Keith-CY

**Comment by Keith-CY** [comment]
> > I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.
> 
> I think this slight difference won't cause confuse, so let‘s approve this PR and merge? @Keith-CY

That's fine

**Review by Keith-CY — APPROVED** [review]

**Comment by homura** [comment]
> I've noticed that `Godwoken Bridge` is mentioned as `GodwokenBridge` in `via {{bridge}} from {{chain}}` while `Force Bridge` is mentioned as `Forcebridge` in the same place, the `B/b` of `bridge` is slightly different.

This is due to the `displayName` of ForceBridge using “Frocebridge” in this [commit](https://github.com/nervosnetwork/force-bridge/pull/373/files#diff-637f871f40326f9de15b973d07082ddf7f0cbcbb03357e676eb7905152cc51caR17), I think godwoken-info should be unified with FB's UAN

File: mainnet_v1/bridged-token-list.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "erc20Info": {
       "sudtScriptArgs": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "ethAddress": "0x7538C85caE4E4673253fFd2568c1F1b48A71558a"
-    }
+    },
+    "UAN": "pCKB.gw|gb.ckb",
```
Comment (Keith-CY) [inline mainnet_v1/bridged-token-list.json:12]: Should we name it `CKB.gw|gb.ckb` since the bridged token is `CKB` but not `pCKB`

Comment (Flouse) [reply mainnet_v1/bridged-token-list.json:12]: The decimals of CKB is 8, while the decimals of pCKB is 18.

**Pull request merged**